cypress/cypress.config.js

```diff
@@ -0,0 +1,53 @@
+const dotenv = require('dotenv')
+const { defineConfig } = require("cypress");
+const browserify = require("@badeball/cypress-cucumber-preprocessor/browserify");
+const { addCucumberPreprocessorPlugin } = require("@badeball/cypress-cucumber-preprocessor");
+
+// Test persistent(between commands) store
+const testStore = {}
+
+async function setupNodeEvents(on, config) {
+  await addCucumberPreprocessorPlugin(on, config);
+
+  on("file:preprocessor", browserify.default(config));
+
+  on("task", {
+    pushValue({ name, value }) {
+      testStore[name] = value
+      return true
+    },
+    getValue(name) {
+      console.log("getValue",name,testStore)
+      return testStore[name]
+    },
+  });
+
+  on("after:run", (results) => {
+    if (results) {
+      console.log(results.status);
+    }
+  });
+
+  return config;
+}
+
+// Import backend .env (smart)?
+const { parsed } = dotenv.config({ path: '../backend/.env' })
```

Comment:
Research, if there is a better practice for Cypress wil be covered in issue  #6374.